code/API_definitions/consent-info.yaml

```diff
@@ -3,32 +3,38 @@ info:
   title: Consent Info API
   description: |
 
-    The Consent Info API allows the API Consumer to verify the Consent status of a particular User for that API Consumer and for the requested scope(s) and Purpose. If the Consent is required for the requested scope(s) and Purpose, but has not been granted by the User, the API can also provide a URL pointing to the API Provider's Consent capture channel, enabling the API Consumer to request Consent from the User.
+    The Consent Info API allows an API Consumer to verify the Consent status of a particular User for that API Consumer (identified by `client_id` from the access token) concerning requested scope(s) and a specific Purpose. It helps determine if Consent is legally required for these parameters. If Consent is necessary but not yet granted, or if it has expired, the API can provide a URL pointing to the API Provider's Consent capture channel. This enables the API Consumer to direct the User to grant or renew Consent in a secure and trusted environment.
+
+    While verifying User Consent is a primary use case, this API allows to check the validity of *any* Legal Basis (e.g., `dpv:Consent`, `dpv:LegitimateInterest`, `dpv:Contract`, etc) that authorizes data processing for a given Purpose.
 
     # Introduction
 
+    The primary function of this API is to enable API Consumers to ascertain the authorization status for processing personal data before using other APIs. By providing a standardized way to check the validity of a legal basis, this API facilitates compliance with privacy regulations and promotes transparency with users. Key inputs include the specific scope(s) of access being requested, the Purpose for data processing, and an identifier for the User (either explicitly provided or derived from a three-legged access token). The API responds with the validity status of the applicable Legal Basis, and, if the Legal Basis is `dpv:Consent`, can provide a URL for the User to manage it.
```

Comment:
Since this API is mainly to provide info to the API consumers as stated in the previous line, not sure how it promotes transparency with users? If I understand it right, we don't mean API consumers = users.